saleor/static/dashboard-next/home/components/HomeProductListCard/HomeProductListCard.tsx

```diff
@@ -0,0 +1,43 @@
+import Card from "@material-ui/core/Card";
+import { withStyles } from "@material-ui/core/styles";
+// import Container from "../../../components/Container";
+import * as React from "react";
+import CardTitle from "../../../components/CardTitle";
+// import PageHeader from "../../../components/PageHeader";
+import i18n from "../../../i18n";
+// import CardSpacer from "../../../components/CardSpacer";
+import HomeProductList from "./HomeProductList";
+
+interface MoneyType {
+  amount: number;
+  currency: string;
+}
+interface HomeProductListCardProps {
+  disabled: boolean;
+  topProducts: Array<{
+    id: string;
+    name: string;
+    orders: number;
+    price: MoneyType;
+    thumbnailUrl: string;
+    variant: string;
+  }>;
+  onRowClick: (id: string) => () => void;
+}
+
+const decorate = withStyles({});
+const HomeProductListCard = decorate<HomeProductListCardProps>(
```

Comment:
Decorate is noop here as it provides no classes. Please use `const HomeProductListCard: React.StatelessComponent<HomeProductListCardProps> = () => <... />` definition instead

Reply:
Also, this component is unnecessary, because it only wraps `<HomeProductList />` component in card. It provides no profit from composition